.github/scripts/setup-env.sh

```diff
@@ -33,43 +32,15 @@ if [[ "${OS_TYPE}" == "macos" && $(uname -m) == x86_64 ]]; then
 fi
 echo '::endgroup::'
 
-echo '::group::Set PyTorch conda channel and wheel index'
```

Comment:
This whole group only set the right conda channel and wheel index to install PyTorch from. Since we don't do the former and the latter is replaced by the `PIP_INSTALL_TORCH` variable generated by the job itself, we no longer need it.